src/layouts/InterfaceLayout/containers/NFTManagerContainer/NFTManagerContainer.vue

```diff
@@ -0,0 +1,493 @@
+      </nft-side-menu>
+      <div v-if="showDetails">
+        <nft-details
+          :nft="detailsFor"
+          :selected-title="nftTitle"
+          @back="comeBack"
+        ></nft-details>
+      </div>
+      <div v-if="!showDetails">
+        <content-block-title :button-text="ntfCount" :title="nftTitle" />
+        <!-- TODO Clean Up Design and Improve Mobile -->
+        <div class="grid-container">
+          <div v-for="nft in nftToShow" :key="nft.key" class="kitty">
+            <div class="kitty-img" @click="showNftDetails(nft)">
+              <div v-show="!hasImage(nft)" class="placeholder">
+                <div class="animated-background"></div>
+              </div>
+              <div v-show="hasImage(nft)">
+                <img :src="getImage(nft)" />
+              </div>
+              <p>#{{ nft.token }}</p>
+            </div>
+          </div>
+          <div v-show="selectedNtf.count > 9" class="internal-nav-container">
+            <span
+              v-show="startIndex > 0"
+              class="internal-nav prev"
+              @click="getPrevious()"
+            >
+              <i class="fa fa-chevron-left"></i>
+            </span>
+            {{ $t('dapps.showing', { first: startIndex, last: endIndex }) }}
+            <!--            Showing {{ startIndex }} -{{ endIndex }}-->
+            <span
+              v-show="showNextButton"
+              class="internal-nav next"
+              @click="getNext()"
+            >
+              <i class="fa fa-chevron-right"></i>
+            </span>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="flex--row--align-start mft-manager-content-container"></div>
+  </div>
+</template>
+
+<script>
+import InterfaceContainerTitle from '@/layouts/InterfaceLayout/components/InterfaceContainerTitle';
+import ContentBlockTitle from '@/layouts/InterfaceLayout/components/ContentBlockTitle';
+import NFTSideMenu from '@/layouts/InterfaceLayout/containers/NFTManagerContainer/components/NFTSideMenu';
+import NftDetails from './containers/NftDetails';
+import { mapState } from 'vuex';
+import hexDecoder from './binaryDecoderNFT';
+import { nftABI } from './abis';
+import fetch from 'node-fetch';
+
+const URL_BASE = 'https://swap.mewapi.io/nft';
+
+export default {
+  components: {
+    'content-block-title': ContentBlockTitle,
+    'nft-side-menu': NFTSideMenu,
+    'interface-container-title': InterfaceContainerTitle,
+    'nft-details': NftDetails
+  },
+  props: {},
+  data() {
+    return {
+      nftABI,
+      countPerPage: 9,
+      nftConfig: [],
+      tokenHelper: {},
+      imagesRetrieved: true,
+      showDetails: false,
+      selectedContract: '0x06012c8cf97bead5deae237070f9587f8e7a266d',
+      detailsFor: {},
+      nftTokens: {},
+      nftData: {},
+      ownedTokens: [],
+      tokenContractAddress: '0xeA3352C1a3480Ac5a32Fcd1F2854529BA7193F14',
+      useDevAddress: false, // todo remove dev item
```

Comment:
please remove all references to `useDevAddress`, you should simply have a global accountAddress object which you can switch between `devAddress` and localAccount